internal/models/application.go

```diff
@@ -70,6 +70,7 @@ type Application struct {
 	Is18OrOlder          bool
 	MLHCodeOfConduct     bool
 	MLHContestAndPrivacy bool
+	CheckedInStatus      bool
```

Comment:
This struct should reflect the table, in this case the application table. So when adding in a new attribute(s) , make sure its reflected here as well.